### ISSUE TITLE
Need to fix the order of the cards

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jsvazic/double-fanucci-ga#6]
```
The current order of the Fanucci cards is not correct.  It should be:

- Bugs
- Lamps
- Mazes
- Fromps
- Hives
- Inkblots
- Ears
- Time
- Scythes
- Zurfs
- Books
- Plungers
- Tops
- Rain
- Faces
```

Original issue reported on code.google.com by `jsva...@gmail.com` on 8 May 2009 at 8:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `jsva...@gmail.com` on 9 May 2009 at 3:18
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed for v1.1
```

Original comment by `jsva...@gmail.com` on 9 May 2009 at 5:13
- Changed state: **Fixed**
